examples/bank_account/bank_account.py

```diff
@@ -26,7 +26,7 @@ def balance(self):
     def withdraw(self, amount):
         """ withdraw money """
         if self._balance >= amount > 0:
-            self._balance -= (amount + self._calc_commission_rate(self._hasCommissionDiscount))
+            self._balance += (amount + self._calc_commission_rate(self._hasCommissionDiscount))
```

Comment:
Revert the changes in the deposit and withdraw methods. The deposit method should increase the balance and the withdraw method should decrease the balance. [important]